Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/ActionManager.cpp

```diff
@@ -420,6 +420,20 @@ namespace AzToolsFramework
         return AZ::Success();
     }
 
+    int ActionManager::GenerateActionAlphabeticalSortKey(const AZStd::string& actionIdentifier)
+    {
+        auto actionIterator = m_actions.find(actionIdentifier);
+        if (actionIterator == m_actions.end())
+        {
+            return AZStd::numeric_limits<int>::max();
```

Comment:
suggestion: Returning `AZStd::numeric_limits<int>::max();` seems like a sentinel value to me 😝 Might this be better to use an optional instead so users clearly know if there's a valid value or not?

Reply:
Returning an optional would mean that each individual caller would have to handle the case where no value is returned differently, causing fragmentation in behavior. I would prefer to provide a default somehow.
I returned max so that items would default to last, but it's more of a way to allow for more complex behavior in the future than anything.

Reply:
Okay understood, if this is just last and the correct behavior will happen if not checked for that's fine, I just wanted to double check, thanks for the additional context